docs/building-apps/wallet/sep24.mdx

```diff
@@ -236,6 +324,13 @@ It's also possible to fetch transaction by the asset
 val transactions = sep24.getTransactionsForAsset(asset, token)
 ```
 
+```typescript
+const transactions = await anchor.getHistory({
```

Comment:
`getHistory` should be deprecated... It has some extra filtering logic that shouldn't be there.
`getTransactionsForAsset` is more flexible than `getHistory`